cider/tests/utils/ArrowArrayBuilderTest.cpp

```diff
@@ -94,6 +94,20 @@ TEST(ArrowArrayBuilderTest, RowNum) {
       CiderCompileException);
 }
 
+TEST(ArrowArrayBuilderTest, BoolColumnBatch) {
```

Comment:
I suggest to add more boundary condition like bit length cannot align to 8. (maybe we can add more when checker part ready)